README.md

```diff
@@ -4,70 +4,90 @@
 
 # Knife CLC
 
-This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like datacenters, templates and groups.
+This is the Chef Knife plugin for CenturyLink Cloud. It gives Knife the ability to manage servers and query for additional resources like data centers, templates and groups.
+
+## Requirements
 
 ## Installation
 
-If you're using [ChefDK](https://downloads.chef.io/chef-dk/), simply install the Gem:
+There are three ways to install the Chef Knife plugin on your CenturyLink Cloud platform.
+
+### If you're using [ChefDK](https://downloads.chef.io/chef-dk/):
```

Comment:
I would also revert those headings too. Take a look at the structure of READMEs from other plugins:
https://github.com/chef/knife-ec2
https://github.com/chef/knife-google
https://github.com/chef/knife-openstack